modin/engines/base/frame/data.py

```diff
@@ -234,6 +246,64 @@ def _filter_empties(self):
         self._column_widths_cache = [w for w in self._column_widths if w > 0]
         self._row_lengths_cache = [r for r in self._row_lengths if r > 0]
 
+    def _validate_axis_equality(self, axis, update):
+        internal_axis = self._frame_mgr_cls.get_indices(
+            axis, self._partitions, lambda df: df.axes[axis]
+        )
+        is_equals = self.axes[axis].equals(internal_axis)
+        if not update:
+            ErrorMessage.catch_bugs_and_request_email(
+                not is_equals, "Internal and external indices do not match.",
+            )
+        elif not is_equals:
+            self._set_axis(axis, self.axes[axis])
+
+    def _validate_internal_indices(self, mode=None, **kwargs):
+        """
+        Validates and optionally updates internal and external indices
+        of modin_frame in specified mode. There is 3 modes supported:
+            1. "reduced only" - validates and updates indices on that axes
```

Comment:
Instead of `"reduced only"` can we use `"reduced"` or something similar? (just a single word is preferred).

Reply:
changed it to `"reduced"`